defaults/k8s_local_kind_config.yaml

```diff
@@ -0,0 +1,33 @@
+instance_provision: 'on_demand'
+
+minikube_version: '0.11.0'
+
+n_db_nodes: 3
+
+scylla_version: '4.2.0'
+scylla_mgmt_agent_version: '2.3.0'
+mgmt_docker_image: 'scylladb/scylla-manager:2.3.0'
+
+# NOTE: If 'k8s_scylla_operator_docker_image' not set then the one from helm chart will be used.
+# To test nightly builds define it like this: 'scylladb/scylla-operator:nightly'
+k8s_scylla_operator_docker_image: ''
+k8s_scylla_operator_helm_repo: 'https://storage.googleapis.com/scylla-operator-charts/latest'
+k8s_scylla_operator_chart_version: 'latest'
+k8s_cert_manager_version: '1.2.0'
+
+k8s_scylla_datacenter: 'gce'
+k8s_scylla_rack: 'minikube'
```

Comment:
kind

Reply:
"minikube" and "kind" are names of 2 different projects. So, we must use here the proper project name.

Reply:
@vponomaryov , yap, that is why we change it to kind in kind's default configuration file

Reply:
fixed